app/views/tasks/show.erb

```diff
@@ -1,20 +1,29 @@
 
 <section id="task_list">
-<h2>Task</h2>
```

Comment:
Watch you indentation here - you'll definitely want to tab in whenever you open a tag (specifically here, after the section tag)